File: C_Programming/Practice/program2.c
// Program to perforrm addition of two integers

#include<stdio.h>

int main()
{
    int iValue1 = 0;
    int iValue2 = 0;
    int iRet = 0;

    printf("Enter first number:");
    scanf("%d",&iValue1);

    printf("Enter second number:");
    scanf("%d",&iValue2);

    iRet = iValue1 + iValue2;

    printf("Addition is : %d",iRet);

    return 0;
}

File: Java_Programming/Practice/program2.java
// Program to perforrm addition of two integers

import java.util.Scanner;

class Addition
{

    public static void main(String A[])
    {
        int iValue1 = 0;
        int iValue2 = 0;
        int iRet = 0;

        Scanner sc = new Scanner(System.in);

        System.out.print("Enter first number:");
        iValue1 = sc.nextInt();

        System.out.print("Enter second number:");
        iValue2 = sc.nextInt();

        iRet = iValue1 + iValue2 ;

        System.out.println("Addition is:"+iRet);


    }
}

